requirements/common.txt

```diff
@@ -26,6 +26,10 @@ backports.ssl-match-hostname==3.5.0.1
 
 # Needed for S3 file uploads
 boto==2.45.0
+boto3==1.4.4
```

Comment:
we require boto because there is still no equivalent of boto.utils.get_instance_metadata() in boto3.
https://github.com/boto/boto3/issues/313